Documentation/project-docs/developer-guide.md

```diff
@@ -3,6 +3,12 @@ Developer Guide
 
 This guide provides instructions (mostly as links) on how to build the repo and implement improvements. It will expand over time.
 
+Dev Workflow
+===============
+
+The [dev workflow] (../building/dev-workflow.md) describes the development process to follow when working in our repo. 
+It is divided into specific tasks that are fast, transparent and easy to understand.
+
 Building the repository
```

Comment:
I thought about doing that, but because we haven't turned on the Dev workflow, wouldn't it be confusing to ask people to run build.cmd with various variables that "enable" the workflow? and once we turn it on ask people to just run the scripts?